### DISCUSSION TITLE
Image quality not the best

### DISCUSSION THREAD

**Discussion by tahadnan** [fastfetch-cli/fastfetch#983] [Q&A] [answered]
Hello, first I want to thank everyone that has contirbuted in this project it is functioning like charm, but this little problem bothers me, the picture quality seems a little bit low 
![image](https://github.com/fastfetch-cli/fastfetch/assets/110527009/21be268c-b282-4695-af07-691d56732f7d)
here is my config: 
[config.jsonc](https://github.com/fastfetch-cli/fastfetch/files/15505356/config.jsonc)
Any help or even a suggestion would be much appreciated

**Comment by CarterLi** [comment, marked as answer]
Please try `{ "type": "iterm" }`, remove `width` and `height` and resize the image yourself.

If it doesn't work, please upload the original image file

**Reply by tahadnan** [reply]
I switched the picture to this one 
![image](https://github.com/fastfetch-cli/fastfetch/assets/110527009/9ac520e7-d5e6-4e76-a928-6e87ccdf0669)
and it's working just fine thanks for the good work all love <3
